README.md

```diff
@@ -243,32 +64,22 @@ Cubit>
 
 If this output does not appear, then the plugin was not installed correctly.
```

Comment:
this is not true on MacOS with Cubit 2021.5

Reply:
here is my command input:
```
Finished Command: undo on

Cubit>help dagmc

Help for words: dagmc.

export dagmc <filename> [faceting_tolerance <faceting tolerance>] [length_tolerance <length tolerance>]
     [normal_tolerance <normal tolerance>] [make_watertight] [verbose]
     [fatal_on_curves]


Help for words: dagmc.

export dagmc <filename> [faceting_tolerance <faceting tolerance>] [length_tolerance <length tolerance>]
     [normal_tolerance <normal tolerance>] [make_watertight] [verbose]
     [fatal_on_curves]

Cubit>
```

not `Loaded Svalinn plugin` input but `help dagmc` worked just fine

Reply:
ok I can perhaps change it to something like ...


Run ```coreform_cubit``` from the command line. After the Cubit GUI finishes loading, run the following command in the Cubit command line:

```bash
help dagmc
```

If the plugin was installed correctly you should see the following text appear.

```
Help for words: dagmc.

export dagmc <filename> [faceting_tolerance <faceting tolerance>] [length_tolerance <length tolerance>]
     [normal_tolerance <normal tolerance>] [make_watertight] [verbose]
     [fatal_on_curves]
```

If this output does not appear, then the plugin was not installed correctly.